### PR TITLE
SSL message removed when server is local. (#15584)

### PR DESCRIPTION
**Pull request by Jaideep-97**
Signed-off-by: JAIDEEP DATTA <34120588+Jaideep-97@users.noreply.github.com>

### Description

SSL message removed when server is local.

Fixes #

15584

### CODE REVIEW

File: templates/home/index.twig [hunk 1]
```diff
@@ -115,10 +115,13 @@
                 {% trans 'Server type:' %}
                 {{ database_server.type }}
               </li>
-              <li id="li_server_connection">
-                {% trans 'Server connection:' %}
-                {{ database_server.connection|raw }}
-              </li>
+
+          {%  if(database_server.host !="Localhost via UNIX socket") %}
```
Comment (ibennetch) [inline templates/home/index.twig]: I think this indentation looks to be not ideal; I think you should line up the `{%` in line with the `</li`> above it, then one space before `if` (the newline is fine, though).

@mauriciofauth what do you think?

Comment (ibennetch) [reply templates/home/index.twig]: I don't think this test will work due to translations. For example, in German this message is shown as "Server-Verbindung: SSL wird nicht verwendet"

Comment (ibennetch) [reply templates/home/index.twig]: I wonder what would happen with Windows using a named pipe. I don't have a Windows MySQL installation to check what message is shown, but we could hide the message in the case of a named pipe as well as UNIX socket.

File: templates/sql/query.twig [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
-  <input type="hidden" name="prev_sql_query" value="{{ query }}">
+  <input type="hidden" name="prev_sql_query"  value="{{ query }}">
 
   {% if display_tab == 'full' or display_tab == 'sql' %}
-    <a id="querybox"></a>
+    <a id="querybox" ></a>
     <div id="queryboxcontainer">
       <fieldset id="queryboxf">
-        <legend>{{ legend|raw }}</legend>
+        <legend style="font-size:0.7em;">{{ legend|raw }}</legend>
```
Comment (ibennetch) [inline templates/sql/query.twig]: Looks like you've made a few unrelated changes to this file as well, please revert those.

**Review by ibennetch — CHANGES_REQUESTED** [review]
Overall I think this is a good idea.

Your pull request has a lot of extra files that aren't needed as part of this submission, can you remove the files composer-setup.php, composer.phar, and yarn-error.log?

**Comment by williamdes** [comment]
The submitter of the issue now agrees that the text should be black https://github.com/phpmyadmin/phpmyadmin/issues/15584#issuecomment-569140761

Thank you for the contribution.

**Pull request closed without merging**